cdap-distributions/src/COPYRIGHT/babel-plugin-syntax-jsx@6.18.0/COPYRIGHT

```diff
@@ -0,0 +1,35 @@
+# babel-plugin-syntax-jsx
```

Comment:
Does not look like COPYRIGHT

Reply:
good catch, it's generated by scripts, this looks like an UI dependency. Let me see how it looks like for other releases.

Reply:
I know why it looks like this now, that's the whole content of its README file...

Reply:
This link does not show MIT, but could you put it into overrides with a link that clarifies it's MIT?